xc/xenops_server_xen.ml

```diff
@@ -127,11 +127,11 @@ module VmExtra = struct
     let open Domain in
     match vm.ty with
     | PV _ ->
-        X86 {emulation_flags= []}
+        X86 {emulation_flags= []; misc_flags= []}
     | PVinPVH _ ->
-        X86 {emulation_flags= emulation_flags_pvh}
+        X86 {emulation_flags= emulation_flags_pvh; misc_flags= []}
     | HVM _ ->
-        X86 {emulation_flags= emulation_flags_all}
+        X86 {emulation_flags= emulation_flags_all; misc_flags= []}
```

Comment:
To fix nested virt I think we need something like this (untested): https://github.com/edwintorok/xenopsd/commit/8359533796e159b60a0c0c5228f9094e754e9266

Reply:
looks plausible